### DISCUSSION TITLE
feat: use moon icon when toggling dark mode

### DISCUSSION THREAD

**Discussion by kranurag78** [hugo-sid/hugo-blog-awesome#13] [Feature requests]
It will be great if we change the icon to moon when we switch to the dark mode.

**Comment by hugo-sid** [comment]
Nice idea Anurag.

**Comment by hugo-sid** [comment]
Hi Anurag. In order to organize the repo in a better manner, I have decided to move all feature request related issues to discussions.

I believe that Discussions will provide a better platform for us to gather feedback and discuss a feature request in a more collaborative manner.

You can find your feature request now in Discussions. I have migrated comments and discussion threads from the original GitHub issue to the new Discussions thread.
